_data/icons/lukso.json

```diff
@@ -1,8 +1,8 @@
 [
   {
-    "url": "ipfs://Qmeg9sFF5tAGi6MCx7YjtVHW6a23zqvHRK1xwzSdp9iE7z",
```

Comment:
Please delete the old instance of `Qmeg9sFF5tAGi6MCx7YjtVHW6a23zqvHRK1xwzSdp9iE7z` in `iconsDownload`